ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1436,6 +1436,10 @@ namespace OrdinalAxis {
 
             // When more series assign to axis, find the smallest one, #15987.
             axis.series.forEach((series): void => {
+                // Don't include indicators and invisible series
+                if (!series.visible || series.is('sma')) {
```

Comment:
Why are indicators out? Some indicators, for example IKH, adds more points at the end of the chart (points are in the future, compared to the linked to series). Why PivotPoints indicator causes this?

Reply:
it is for finding the first point, which is in `ordinalPositions`. `pivotPoints` is a series, so it has points, but it is more like annotation, so its points don't matter alone. When it comes to IKH, it adds the points at the end, so it shouldn't be an issue, but I will test it out. 

Maybe the whole logic is to be rewritten? :D

Reply:
It looks like we are hiding the real culprit here, instead of finding the issue's root. Indicators itself shouldn't be a problem. If we replace PivotPoints with a scatter series we get the same issue: https://jsfiddle.net/BlackLabel/6mfwdop7/2/ (I just copied x-values from pivot points and set y=200 for each point).